# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/crashtests/1965664.html
# Commit: 3aa74b745aa5
# Full Hash: 3aa74b745aa5ae1f1b460452f223ad0b9633fec0
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-05-13 09:26:34
# Description:
#   Bug 1965664 - Fix some crashes in ContentSubtreeIterator r=jjaschke,dom-core
#   
#   by skipping the shadow hosts that ShadowDOM selection doesn't
#   support at the moment.
#   
# ==============================================================================

diff -r ecccb8942de4 -r 3aa74b745aa5 dom/base/crashtests/1965664.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/base/crashtests/1965664.html	Mon May 12 20:40:53 2025 +0000
@@ -0,0 +1,10 @@
+<script>
+addEventListener("DOMContentLoaded", () => {
+  a.ownerDocument.designMode = "on"
+  document.execCommand("selectAll", false, null)
+  document.execCommand("superscript", false, null)
+})
+</script>
+<s>
+<h2>
+<marquee id="a">